### ISSUE TITLE
[Suggestion]: Transform the statues into vanilla armor stands

### ISSUE THREAD

**Issue by BeeFox-sys** [Fuzss/strawstatues#25]
### Suggestion (Required)

I would love to use this mod, however i would like to be able to transform them into regular armor stands (maybe with a custom tag) and back, in order to be able to load the world on versions incompatable with the mod, but keep at least a one way transformation into armor stands so that they still exist in the world.

If there is some way to store the data on the armor stand in a way compatable with vanilla, so that it can also be transformed back, that would be amazing!

To summarize: A command to turn a straw statue into an armorstand, ideally maintaining the data for things like skin and etc that armor stands do not support, so that they may be transformed to (and hopefully from) a vanilla friendly format

**Comment by Fuzss** [comment]
Thanks for the suggestion!

This is already possible to some extend as of Minecraft 1.20 (not possible in 1.19.2 at the moment):
I also have a mod called [Armor Statues](https://www.curseforge.com/minecraft/mc-mods/armor-statues) which implements the interface used for straw statues for vanilla armor stands. Install that mod in addition to Straw Statues.
Then on the rotations screen there are copy and paste buttons, those work across both mods, so you can copy your straw statue rotations and apply them to a vanilla armor stand.

This is the farthest I'll support such a conversion in the mod directly.

**Comment by BeeFox-sys** [comment]
That helps, thanks c:
